File: 백준/Silver/1475. 방 번호/방 번호.java
import java.util.*;
import java.io.*;

public class Main{
    public static void main(String[] args) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(br.readLine());

        // 카드 넣을 배열 생성
        int[] arr = new int[10];
        // 9인경우 6에 ++
        while(n != 0){
            if(n % 10 == 9) arr[6]++;
            else arr[n % 10]++;
            n /= 10;

        }

        arr[6] = (int)Math.ceil((double)arr[6] / 2);

        // 배열 돌면서 가장 큰 개수가 카드의 개수
        int max = 0;


        for(int i : arr) {
            if (max < i) {
                max = i;
            }
        }

        System.out.println(max);
    }
}

File: 백준/Silver/1475. 방 번호/방 번호.py
room = input()

arr = [0] * 10

for c in room :
    if c == '6' or c == '9' :
        arr[6] += 1
    else :
        arr[int(c)] += 1

arr[6] = (arr[6] + 1) // 2

max = arr[0]

for i in range(1, 10) :
    if max < arr[i] :
        max = arr[i]

print(max)